Lib/pydoc.py

```diff
@@ -1574,7 +1574,8 @@ def getpager():
     if sys.platform == 'win32':
         return lambda text: tempfilepager(plain(text), 'more <')
     if hasattr(os, 'system') and os.system('(less) 2>/dev/null') == 0:
-        return lambda text: pipepager(text, 'less')
+        cmd = 'less "-P?e(END) .(q to quit)" --quit-at-eof'
```

Comment:
Do also considering adding the `-X` and `-F` (latter is the same as `--quit-if-one-screen`) flags.

From the less documentation,

`-X` causes less to not clear screen upon quit

>     -X or --no-init
> 
>     Disables sending the termcap initialization and deinitialization strings to the terminal. This is sometimes desirable if the deinitialization string does something unnecessary, like clearing the screen.

and with -F, it quits if the help fits in one screen

>     -F or --quit-if-one-screen
> 
>     Causes less to automatically exit if the entire file can be displayed on the first screen.

```suggestion
        cmd = 'less "-P?e(END) .(q to quit)" -X --quit-if-one-screen --quit-at-eof'
```